src/components/VmsList/Vms.js

```diff
@@ -37,11 +37,12 @@ class Vms extends React.Component {
       return vmAName.localeCompare(vmB.get('name'))
     }
 
-    const sortedVms = vms.get('vms').toList().sort(sortFunction)
+    const sortedVms = vms.get('vms').toList().map(vm => vm.set('isVm', true))
     const sortedPools = vms.get('pools')
-      .filter(pool => (pool.get('vmsCount') < pool.get('maxUserVms') && pool.get('size') > 0))
+      .filter(pool => (pool.get('size') > 0))
       .toList()
-      .sort(sortFunction) // TODO: sort vms and pools together!
+
+    const vmsPoolsMerge = [ ...sortedVms, ...sortedPools ].sort(sortFunction)
```

Comment:
Maybe pull the `sortFunction` out of the `render` function scope to make sure it won't get a new context redefined with every `render` call?